File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/zig/F07.P1.zig
const std = @import("std");

var src: i32 = 6;

fn probe() i32 {
    const a: i32 = src;
    const b: i32 = 7;
    const c: i32 = 5;
// BEGIN PROBE F07.P1
const r = a * b + c;
// END PROBE F07.P1
    return r;
}

pub fn main() !void {
    var threaded: std.Io.Threaded = .init_single_threaded;
    const io = threaded.io();
    var out_buf: [64]u8 = undefined;
    var out = std.Io.File.stdout().writer(io, &out_buf);
    try out.interface.print("{d}\n", .{probe()});
    try out.interface.flush();
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/java/F07.P1.java
class Main {
    static int probe(int a, int b, int c) {
        // BEGIN PROBE F07.P1
        int r = a * b + c;
        // END PROBE F07.P1
        return r;
    }

    public static void main(String[] args) {
        System.out.println(probe(2, 3, 4));
    }
}
